Yosemite/Yosemite/Actions/ProductAction.swift

```diff
@@ -14,7 +14,15 @@ public enum ProductAction: Action {
     ///
     case retrieveProduct(siteID: Int, productID: Int, onCompletion: (Product?, Error?) -> Void)
 
+    /// Retrieves a specified list of Products.
+    ///
+    case retrieveProducts(siteID: Int, productIDs: [Int], onCompletion: (Error?) -> Void)
+
     /// Deletes all of the cached products.
     ///
     case resetStoredProducts(onCompletion: () -> Void)
+
+    /// Synchronizes the Products found in a specified Order.
+    ///
+    case synchronizeProductsFor(_ order: Order, onCompletion: (Error?) -> Void)
```

Comment:
I would make the "for" the parameter name, it reads a bit better to me:
```
case synchronizeProducts(for order: Order, onCompletion: (Error?) -> Void)
```

But that's just me, this is a completely subjective comment that can be totally ignored :D

Reply:
I tried that earlier but Xcode yelled at me and said "enum case cannot have reserved keyword 'for' "

Reply:
It bugs me too. I'll think of a better method name.

Reply:
Can't have enums with the exact same case name and different tuples :(